backend/api/quivr_api/models/settings.py

```diff
@@ -125,6 +121,7 @@ class BrainSettings(BaseSettings):
     langfuse_secret_key: str | None = None
     pg_database_url: str
     pg_database_async_url: str
+    embedding_dim: int = int(os.getenv("EMBEDDING_DIM", 1536))
```

Comment:
The BrainSettings in a SettingsConfiDict. It automatically resolves values from .env and environement variable:
https://docs.pydantic.dev/latest/concepts/pydantic_settings/


So no need to set it here 👍